R/03-geojob-obj.R

```diff
@@ -110,7 +110,7 @@ setMethod(f = "xml<-",signature = "geojob", definition = function(.Object, value
 #'@rdname geojob-methods
 #'@aliases xml,geojob-method
 #'@examples
-#'xml <- "<foo> <bar> text <baz/> </bar> </foo>"
+#'xml <- "<foo version=\"1.0.0\"> <bar> text <baz/> </bar> </foo>"
```

Comment:
I didn't think there would be a requirement for what the xml contains in that slot, just that it is xml...